test/odbc/psqlodbc/float.cpp

```diff
@@ -0,0 +1,458 @@
+#include "psqlodbc_tests_common.h"
+
+const string TABLE_NAME = "master_dbo.float_table_odbc_test";
+const string VIEW_NAME = "master_dbo.float_view_odbc_test";
+const string DATATYPE_NAME = "sys.float";
+const string COL1_NAME = "pk";
+const string COL2_NAME = "data";
+
+const vector<pair<string, string>> TABLE_COLUMNS = {
+  {COL1_NAME, "int PRIMARY KEY"},
+  {COL2_NAME, DATATYPE_NAME}
+};
+
+const string FLOAT_15 = "1e308";
+const string FLOAT_383 = "-1e307";
+
+const int FLOAT_BYTES_EXPECTED = 8;
+
+class PSQL_DataTypes_Float : public testing::Test {
+
+  void SetUp() override {
+    if (!Drivers::DriverExists(ServerType::PSQL)) {
+      GTEST_SKIP() << "PSQL Driver not present: skipping all tests for this fixture.";
+    }
+    OdbcHandler test_setup(Drivers::GetDriver(ServerType::PSQL));
+    test_setup.ConnectAndExecQuery(DropObjectStatement("TABLE", TABLE_NAME));
+  }
+  
+
+  void TearDown() override {
+    if (!Drivers::DriverExists(ServerType::PSQL)) {
+      GTEST_SKIP() << "PSQL Driver not present: skipping all tests for this fixture.";
+    }
+
+    OdbcHandler test_cleanup(Drivers::GetDriver(ServerType::PSQL));
+    test_cleanup.ConnectAndExecQuery(DropObjectStatement("VIEW", VIEW_NAME));
+    test_cleanup.ExecQuery(DropObjectStatement("TABLE", TABLE_NAME));
+  }
+};
+
+// Helper to convert string to double/float
+double StringToFloat(const string &value) {
+  if (value == "NULL") {
+    return 0;
+  }
+  return atof(value.c_str());
+}
+
+vector<double> getExpectedFloatResults(vector<string> data) {
```

Comment:
nit: Let's keep the naming of functions at least consistent within a file (psql_common and odbchandler have different naming conventions so we can't make it 100% consistent, but lets contain it at-least in the file-level). Either do Pascal or camel casing for `StringToFloat` and `getExpectedFloatResults`.  This goes with the other files.

Reply:
Fixed. I moved all the `getExpected...` and the conversion functions to their own file, `conversion_functions_common`. The function naming should be consistent there.